rocky/reports/views/generate_report.py

```diff
@@ -132,6 +135,12 @@ class SetupScanGenerateReportView(
     current_step = 3
 
     def post(self, request, *args, **kwargs):
+        if "return" in self.request.POST and self.plugins_enabled():
+            return ReportTypesSelectionGenerateReportView().post(request, *args, **kwargs)
```

Comment:
I've rarely seen views being initialized in another view. This also skips the view execution flow the framework (setup, dispatch, render). Is this the desired behavior?